src/components/CatalogOverlay/CatalogSubplotComponent/CatalogSubplotComponent.tsx

```diff
@@ -164,10 +228,19 @@ export class CatalogScatterComponent extends React.Component<WidgetProps> {
             const catalogFileId = this.widgetStore.catalogOverlayWidgetStore.catalogInfo.fileId;
             AppStore.Instance.updateCatalogProfiles(catalogFileId);
             let selectedPointIndices = [];
-            const points = event.points;
+            const points = event.points as any;
             for (let index = 0; index < points.length; index++) {
                 const selectedPoint = points[index];
-                selectedPointIndices.push(selectedPoint.pointIndex);
+                if (this.widgetStore.plotType === CatalogPlotType.D2Scatter) {
+                    selectedPointIndices.push(selectedPoint.pointIndex);   
+                }
+    
+                if (this.widgetStore.plotType === CatalogPlotType.Histogram) {
```

Comment:
might be better to move the if statement out of the loop, and then you can also assign the proper type to `points` instead of `any`

Reply:
the current types we use for plotly is not support the histogram, that's why I use any for it.